index.bs

```diff
@@ -9430,26 +9430,33 @@ The <dfn>own exposure set</dfn> is the [=/set=] of identifiers defined as follow
     1.  Let |H| be |C|'s [=host interface=] if |C| is an [=interface mixin member=], or null otherwise.
     1.  If |C| is an [=interface member=], [=interface mixin member=], or [=namespace member=], then:
         1.  If the [{{Exposed}}] [=extended attribute=] is specified on |C|, then:
-            1.  If |H| is set, return the [=set/intersection=] of |C|'s [=own exposure set=]
-                and |H|'s [=exposure set=].
-            1.  Otherwise, return |C|'s [=own exposure set=].
-        1.  Otherwise, set |C| to be the
+            1.  If |H| is null, return |C|'s [=own exposure set=].
+            1.  If |C|'s [=own exposure set=] is <code>*</code>, return |H|'s [=exposure set=].
+            1.  If |H|'s [=exposure set=] is <code>*</code>, return |C|'s [=own exposure set=].
+            1.  Return the [=set/intersection=] of |C|'s [=own exposure set=] and |H|'s
+                [=exposure set=].
```

Comment:
What do you think of pulling these four lines out into a helper function, e.g. "exposure set intersection of construct |C| and interface-or-null |H|"?

Reply:
I'd be happy to. Any other thoughts?

Reply:
Seems reasonable otherwise!

Reply:
Done.